pm/sendermonitor_test.go

```diff
@@ -206,29 +186,32 @@ func TestQueueTicketAndSignalMaxFloat(t *testing.T) {
 		},
 	}
 	smgr.claimedReserve[addr] = big.NewInt(100)
-	sm := NewSenderMonitor(claimant, b, smgr, rm, 5*time.Minute, 3600, em)
+	sm := NewSenderMonitor(claimant, b, smgr, tm, 5*time.Minute, 3600)
 	sm.Start()
 	defer sm.Stop()
 
 	assert := assert.New(t)
-	require := require.New(t)
 
 	// Test queue ticket
 
 	sm.QueueTicket(addr, defaultSignedTicket(uint32(0)))
+	time.Sleep(20 * time.Millisecond)
 
-	sm.SubFloat(addr, big.NewInt(5))
+	assert.Equal(sm.(*senderMonitor).senders[addr].queue.Length(), int32(1))
 
 	qc := &queueConsumer{}
 	go qc.Wait(1, sm)
 
-	err := sm.AddFloat(addr, big.NewInt(5))
-	require.Nil(err)
+	tm.blockNumSink <- big.NewInt(5)
+	time.Sleep(20 * time.Millisecond)
+	assert.Equal(sm.(*senderMonitor).senders[addr].queue.Length(), int32(0))
 
-	time.Sleep(time.Millisecond * 20)
 	tickets := qc.Redeemable()
 	assert.Equal(1, len(tickets))
-	assert.Equal(uint32(0), tickets[0].SenderNonce)
+	assert.Equal(uint32(0), tickets[0].Ticket.SenderNonce)
```

Comment:
Why not `.SenderNonce`?

Reply:
fixed in 4860c1fe965df76518e4595d9599f7026730805b